### ISSUE TITLE
Encountered unknown tag 'load'

### ISSUE THREAD

**Issue by ZachLiuGIS** [pydanny/cookiecutter-django#278]
I think someone met similar problem before, here is what I got when trying to set up a cookie cutter project.

```python
Traceback (most recent call last):
  File "/usr/local/bin/cookiecutter", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 610, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 590, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 782, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/cookiecutter/cli.py", line 70, in main
    cookiecutter(template, checkout, no_input)
  File "/usr/local/lib/python3.4/site-packages/cookiecutter/main.py", line 102, in cookiecutter
    context=context
  File "/usr/local/lib/python3.4/site-packages/cookiecutter/generate.py", line 204, in generate_files
    generate_file(project_dir, infile, context, env)
  File "/usr/local/lib/python3.4/site-packages/cookiecutter/generate.py", line 114, in generate_file
    tmpl = env.get_template(infile_fwd_slashes)
  File "/usr/local/lib/python3.4/site-packages/jinja2/environment.py", line 791, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.4/site-packages/jinja2/environment.py", line 765, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.4/site-packages/jinja2/loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "/usr/local/lib/python3.4/site-packages/jinja2/environment.py", line 554, in compile
    self.handle_exception(exc_info, source_hint=source)
  File "/usr/local/lib/python3.4/site-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.4/site-packages/jinja2/_compat.py", line 36, in reraise
    raise value.with_traceback(tb)
  File "./{{cookiecutter.repo_name}}/templates/users/user_form.html", line 2, in <module>
  File "/usr/local/lib/python3.4/site-packages/jinja2/environment.py", line 459, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/usr/local/lib/python3.4/site-packages/jinja2/parser.py", line 893, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/usr/local/lib/python3.4/site-packages/jinja2/parser.py", line 875, in subparse
    rv = self.parse_statement()
  File "/usr/local/lib/python3.4/site-packages/jinja2/parser.py", line 136, in parse_statement
    self.fail_unknown_tag(token.value, token.lineno)
  File "/usr/local/lib/python3.4/site-packages/jinja2/parser.py", line 89, in fail_unknown_tag
    return self._fail_ut_eof(name, self._end_token_stack, lineno)
  File "/usr/local/lib/python3.4/site-packages/jinja2/parser.py", line 82, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "/usr/local/lib/python3.4/site-packages/jinja2/parser.py", line 51, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'load'.
  File "./{{cookiecutter.repo_name}}/templates/users/user_form.html", line 2
    {% load crispy_forms_tags %}
```

**Comment by jayfk** [comment]
Seems this bug was introduced here: https://github.com/pydanny/cookiecutter-django/commit/36454d9893278c4326a13332c3cfa56eba1dc6b2

**Comment by pydanny** [comment]
Fixed in 009c4993cc7a9006954c9515986358dce4f40946. Thanks for the catch!
